### ISSUE TITLE
A11y_AzureDataStudioAAE_Azure ADC(Preview)_Azure Configuration_ColorContrast: Color contrast ratio of the dropdown indicator is less than 3:1

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15804]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Keyboard;#WCAG1.4.11;#AINotInScope;#ColorContrastAnalyzer;#A11yWCAG2.1;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till "Connection" and tab to see more button
4. Tab to "New Deployment" and Azure ADC(Preview) and then tab select button
5. Deployment Prerequisite page will open and tab on next "Select from existing Kubernetes cluster" will open
6. Tab till "Azure account" button and see color contrast of dropdown indicator

 
### Actual:
Color contrast ratio of the dropdown indicator is 1.9:1 which is less than 3:1.

Similar issue is present throughout the application where dropdown indicator is there.

### Expected:
Color contrast ratio of the dropdown indicator should be greater than 3:1.

### User Impact:
Low vision users will not able to view dropdown indicator properly.

**Comment by Charles-Gagnon** [comment]
@sspatil123 Are you referring to the dropdown arrows in the input boxes?

![image](https://user-images.githubusercontent.com/28519865/125330043-18763200-e2fb-11eb-8136-42884b18662b.png)

Given the description I believe that's what you're referring to here and so I'm going to close this as that will be fixed by https://github.com/microsoft/azuredatastudio/pull/16073

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (190)](https://user-images.githubusercontent.com/85863543/150974145-55745965-8f2c-4328-8e38-7234df211a3d.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
